cosmos/providers/dbt/core/profiles/bigquery.py

```diff
@@ -40,7 +40,9 @@ def create_profile_vars_google_cloud_platform(
     https://docs.getdbt.com/reference/warehouse-setups/bigquery-setup
     https://airflow.apache.org/docs/apache-airflow-providers-google/stable/connections/gcp.html
     """
-    bigquery_key_file = json.loads(conn.extra_dejson.get("keyfile_dict"))
```

Comment:
@mmenalla thanks for opening this! Can we add a check for those running various versions of Airflow? Something like:

```python
bigquery_key_file = conn.extra_dejson.get("extra__google_cloud_platform__keyfile_dict") or conn.extra_dejson.get("keyfile_dict")
bigquery_key_file = json.loads(bigquery_key_file or "{}")

if not bigquery_key_file:
    raise ValueError("BigQuery key file not found in connection parameters")
```

*Disclaimer* I haven't tested this yet, but the idea is that we should accommodate as many versions of Airflow as possible.

Reply:
@pohek321 I checked and it works so i pushed the latest changes.